### ISSUE TITLE
authentication version should be taken from the info.json

### ISSUE THREAD

**Issue by nicolasfranck** [UniversalViewer/universalviewer#530]
**UV version:**

```
 universalviewer@beta (current v3)
```

**I'm submitting a:**

- [x] bug report
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

The UV requires the application configuration to specify the authentication version.

**Expected behavior:**

The authentication version should be taken from the info.json

The info.json is retrieved anyway, so why is it not used?
Would be happy to contribute ;-)

**Steps to reproduce:**

**Related code:**

```
// insert any relevant code here
```

**Other information:**

**Comment by nicolasfranck** [comment]
See https://github.com/UniversalViewer/universalviewer/pull/556

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.

**Comment by demiankatz** [comment]
Closed due to inactivity; superseded by #556.
